package.json

```diff
@@ -5,7 +5,7 @@
   "repository": "https://github.com/radiantearth/stac-api-spec",
   "license": "Apache-2.0",
   "scripts": {
-    "serve": "redoc-cli serve core/openapi.yaml --watch --options.expandResponses \"200,201,202,203,204\" --options.pathInMiddlePanel true",
+    "serve": "redoc-cli serve $npm_config_class/openapi.yaml --watch --options.expandResponses \"200,201,202,203,204\" --options.pathInMiddlePanel true",
```

Comment:
syntax is now `npm run server --class=core` (where class can be core, ogcapi-features, item-search)

Reply:
Does that work on all major OS (looking at Windows)?

Reply:
It's part of NPM rather than shell, so it should.